chore: remove user tags functionality. (#50051)
sourcegraph/sourcegraph: CHANGELOG.md

### Removed

> - User tags are removed because they are inferior to the preferred feature flags functionality. [#49318](https://github.com/sourcegraph/sourcegraph/pull/49318)

Reviewer: ```suggestion
- User tags are removed in favor of the newer feature flags functionality. [#49318](https://github.com/sourcegraph/sourcegraph/pull/49318)
```
